README.md

```diff
@@ -11,12 +11,17 @@ These interfaces can be used independently or together, depending on the use cas
 
 ## A note of caution
 
+This proposal contains two advanced features, `WeakRefs` and `FinalizationGroups`. Their correct use takes careful thought, and they are best avoided if possible.
```

Comment:
`WeakRef`s and `FinalizationGroup`s
to stay consistent with some other places in this document?